### ISSUE TITLE
About downstream sft

### ISSUE THREAD

**Issue by MlSAKA-MlKOTO** [TsinghuaC3I/UltraMedical#1]
Hi, thank you for sharing your amazing work. I'm trying to reproduce your result but facing some problems. Could you provide more details about the finetuning on these benchmarks?

**Comment by iseesaw** [comment]
Thank you for your interest in our project!

We directly fine-tuned the model with the whole UltraMedical dataset and evaluated it on downstream tasks.
We have outlined the training details on our [model card](https://huggingface.co/TsinghuaC3I/Llama-3-8B-UltraMedical#training-details). Below are the hyperparameters used:

- **Torch type:** bfloat16
- **Epochs:** 3
- **Learning rate:** 2e-5
- **Learning rate scheduler:** Cosine
- **Warmup ratio:** 0.04
- **Max length:** 1024
- **Global batch size:** 128

Specifically, we recommend maintaining the same format for multi-choice QA and the PubMedQA during evaluations as demonstrated in the [model card](https://huggingface.co/TsinghuaC3I/Llama-3-8B-UltraMedical#input-examples).

Let us know if you need further details!

**Comment by MlSAKA-MlKOTO** [comment]
Thanks very much, looking forward to your paper!
